lit/setup-and-operations/install.lit

```diff
@@ -68,6 +68,232 @@ page.
+    so:
+
+    \codeblock{bash}{{{
+    concourse web \
+      --add-local-user myuser:mypass
+    }}}
+  }
+
+  \section{
+    \title{GitHub Auth}{github-auth-config}
+
+    A Concourse server can authenticate against GitHub to take advantage of
+    their permission model and other security improvements in their
+    infrastructure.
+
+    \section{
+      \title{Creating a GitHub application}
+
+      First you need to \link{create an OAuth application on
+      GitHub}{https://github.com/settings/applications/new}.
+
+      The callback URL will be the external URL of your Concourse server with
+      \code{/sky/issuer/callback} appended. For example, Concourse's own CI
+      server's callback URL would be
+      \code{https://ci.concourse-ci.org/sky/issuer/callback}.
+    }
+
+    \section{
+      \title{Configuring the client}
+
+      You will be given a Client ID and a Client Secret for your new
+      application. These will then be passed to the following flags:
+
+      \list{
+        \code{--github-client-id=CLIENT_ID}
+      }{
+        \code{--github-client-secret=CLIENT_SECRET}
+      }
+
+      If you're configuring GitHub Enterprise, you'll also need to set the
+      following flags:
+
+      \list{
+        \code{--github-host=github.example.com}
+      }{
+        \code{--github-ca-cert=/tmp/some-cert}
+      }
+
+      The GitHub Enterprise host should not contain a scheme, or a trailing
+      slash.
+
+      \codeblock{bash}{{{
+      concourse web \
+        --github-client-id CLIENT_ID \
+        --github-client-secret CLIENT_SECRET
+      }}}
+    }
+  }
+
+  \section{
+    \title{CF Auth}{cf-auth-config}
+
+    Cloud Foundry (CF) Auth can be used for operators who wish to authenticate
+    their users configured against their Cloud Foundry instance via the UAA
+    product.
+
+    \section{
+      \title{Creating the client}
+
+      You'll first need to create a client for Concourse in UAA.
+
+      The callback URL will be the external URL of your Concourse server with
+      \code{/sky/issuer/callback} appended. For example, Concourse's own CI
+      server's callback URL would be
+      \code{https://ci.concourse-ci.org/sky/issuer/callback}.
+
+      The client should look something like this, under
+      \link{\code{uaa.clients}}{http://bosh.io/jobs/uaa?source=github.com/cloudfoundry/uaa-release&version=13#p=uaa.clients}:
+
+      \codeblock{yaml}{{
+        concourse:
+          id: my-client-id
+          secret: my-client-secret
+          scope: openid,email,profile,roles
+          authorized-grant-types: "authorization_code,refresh_token"
+          access-token-validity: 3600
+          refresh-token-validity: 3600
+          redirect-uri: https://concourse.example.com/sky/issuer/callback
+      }}
+    }
+
+    \section{
+      \title{Configuring the client}
+
+      You will be given a Client ID and a Client Secret for your new
+      application. These will then be passed to the following flags:
```

Comment:
Similar feedback as above (specify the command, omit/provide context for the codeblock below).